geotrek/signage/tests/test_command_signage.py

```diff
@@ -20,6 +24,25 @@ class SignageCommandTest(TestCase):
     def setUpTestData(cls):
         cls.path = PathFactory.create()
 
+    def _update_gis(self, input_file_path: str, output_file_path: str, new_properties: Dict):
+        '''
+        Utility function that reads a GIS file (GeoPackage or Shapefile), update some properties,
+        then write a new shapefile (typically in /tmp). Useful to test specific property values.
+        Can eventually be moved somewhere else, since it's not specifically related to signages.
+        '''
+
+        with fiona.open(input_file_path) as source:
+            with fiona.open(output_file_path,
+                            mode='w',
+                            crs=source.crs,
+                            driver=source.driver,
+                            schema=source.schema) as dest:
+                for feat in source:
+                    dest.write(fiona.Feature(
+                        geometry=feat.geometry,
+                        properties={**feat.properties, **new_properties}
+                    ))
```

Comment:
I suggest to move this somewhere in geotrek.common.utils.

Reply:
I suppose you mean the whole `_update_gis` method? Indeed as mentioned in the docstring I was not sure where to put it. So I could move it to `geotrek.common.utils.testdata`, and add its test in `geotrek.common.tests.test_utils` (so no coverage downgrade), what do you think?

Reply:
In 09b13b8 I moved it in geotrek.common.tests.utils.py, I hope it's good.